uw-frame-components/portal/main/partials/header.html

```diff
@@ -1,43 +1,45 @@
-                 <span class="icon-bar"></span>
-                 <span class="icon-bar"></span>
-                 <notification-bell mode='mobile-bell' ng-if='!GuestMode'></notification-bell>
-             </button>
-             <div class="header-crest">
-               <a aria-label="{{portal.theme.ariaLabelTitle}}"
-                  class="header-text" ng-href="{{MISC_URLS.rootURL}}"
-                  target="_self"
-                  id="myuw-header">
-                  <div class='header-inner'>
-                    <img ng-src="{{portal.theme.crest}}"
-                         class="hidden-xs crest"
-                         alt="{{portal.theme.crestalt}}">
-                    <span class='title'>{{portal.theme.title}}</span>
-                    <span class="beta-logo">{{portal.theme.subtitle}}</span>
-                  </div>
-               </a>
-             </div>
-              <div class="search-mobile-icon hidden-sm hidden-md hidden-lg">
-                <i class="fa fa-search" ng-click="headerCtrl.toggleSearch()"></i>
-              </div>
-              <div class="search-mobile" ng-show="showSearch">
-                <search class="hidden-sm hidden-md hidden-lg"></search>
-              </div>
-         </div>
-
-         <div class="collapse navbar-collapse" id="bucky-navbar-collapse"  collapse="headerCtrl.navbarCollapsed">
-             <div class="nav navbar-nav visible-xs bucky-nav">
-                  <ul class="list-group sidebar-list" ng-class="{ 'searching' : showSearch}">
+<div ng-controller="PortalHeaderController as headerCtrl">
+  <notification-partials mode="priority" class="hidden-xs"></notification-partials>
+  <nav class="navbar navbar-default navbar-fixed-top" ng-class="{'has-priority-nots': hasPriorityNotifications}" role="banner">
+       <div class="container-fluid">
+           <div class="navbar-header">
+               <button type="button" class="navbar-toggle" ng-click="headerCtrl.toggleMenu()">
+                   <span class="sr-only">Toggle navigation</span>
+                   <span class="icon-bar"></span>
+                   <span class="icon-bar"></span>
+                   <span class="icon-bar"></span>
+                   <notification-partials mode='mobile-bell' ng-if='!GuestMode'></notification-partials>
+               </button>
+               <div class="header-crest">
+                 <a aria-label="{{portal.theme.ariaLabelTitle}}"
+                    class="header-text" ng-href="{{MISC_URLS.rootURL}}"
+                    target="_self"
+                    id="myuw-header">
+                    <div class='header-inner'>
+                      <img ng-src="{{portal.theme.crest}}"
+                           class="hidden-xs crest"
+                           alt="{{portal.theme.crestalt}}">
+                      <span class='title'>{{portal.theme.title}}</span>
+                      <!-- <span class="beta-logo">{{portal.theme.subtitle}}</span> -->
```

Comment:
Hmm, that might have come from rebasing and trying to fix a merge conflict. I'll uncomment.